homeassistant/components/electrolux_group/sensor.py

```diff
@@ -0,0 +1,336 @@
+        )
+        self._value_fn = description.value_fn
+        self._update_attr_state()
+
+    def _update_attr_state(self) -> None:
+        self._attr_native_value = self._get_value()
+
+    def _get_value(self) -> Any:
+        return self._value_fn(self._appliance_data)
+
+
+class ElectroluxTemperatureSensor(ElectroluxSensor):
+    """Representation of a temperature sensor for Electrolux appliances."""
+
+    def __init__(
+        self,
+        appliance_data: ApplianceData,
+        coordinator: ElectroluxDataUpdateCoordinator,
+        description: ElectroluxSensorDescription,
+    ) -> None:
+        """Initialize the sensor."""
+        super().__init__(appliance_data, coordinator, description)
+        self._attr_device_class = SensorDeviceClass.TEMPERATURE
+        self._attr_state_class = "measurement"
+        self._appliance = cast(OVAppliance | CRAppliance, appliance_data)
+        self._attr_native_unit_of_measurement = ELECTROLUX_TO_HA_TEMPERATURE_UNIT.get(
+            self._appliance.get_current_temperature_unit()
+        )
+        self._update_attr_state()
+
+    def _update_attr_state(self) -> None:
+        self._attr_native_value = self._get_value()
+
+    def _get_value(self) -> Any:
+        return self._value_fn(self._appliance_data)
+
+
+class ApplianceAlertSensor(ElectroluxBaseEntity[ApplianceData], SensorEntity):
+    """Representation of appliance alerts."""
+
+    def __init__(
+        self,
+        appliance_data: ApplianceData,
+        coordinator: ElectroluxDataUpdateCoordinator,
+    ) -> None:
+        """Initialize the alerts sensor."""
+        super().__init__(appliance_data, coordinator)
+        self._attr_name = "Alerts"
+        self._attr_icon = "mdi:alert-circle-outline"
+        self._attr_unique_id = f"{appliance_data.appliance.applianceId}_alerts"
+        self._attr_device_class = SensorDeviceClass.ENUM
+        self._appliance = cast(
+            WMAppliance
+            | WDAppliance
+            | TDAppliance
+            | DWAppliance
+            | OVAppliance
+            | SOAppliance
+            | CRAppliance
+            | HDAppliance
+            | HBAppliance,
+            appliance_data,
+        )
+        self._update_attr_state()
+
+    def _update_attr_state(self) -> None:
+        alerts = self._get_value()
+        if alerts:
+            self._attr_native_value = f"{len(alerts)} alert(s)"
+            self._attr_extra_state_attributes = {
+                f"alert_{i + 1}": {
+                    "code": alert["code"],
+                    "severity": alert["severity"],
+                    "acknowledge_status": alert["acknowledgeStatus"],
+                }
+                for i, alert in enumerate(alerts)
+            }
+        else:
+            self._attr_native_value = "No alerts"
+            self._attr_extra_state_attributes = {}
+
+    def _get_value(self) -> Any:
+        return self._appliance.get_current_alerts()
```

Comment:
I would opt to have this one for a follow-up as well. I think it's good to keep in mind that we want to avoid using extra state attributes as they are hard to automate with. But I am happy to discuss this with you, but I don't want to block the rest of the PR with this